cpp/src/io/orc/reader_impl.cu

```diff
@@ -822,8 +822,8 @@ void update_null_mask(cudf::detail::hostdevice_2dvector<gpu::ColumnDesc>& chunks
     }
   }
 
-  thrust::counting_iterator<int, thrust::host_space_tag> col_idx_it(0);
-  thrust::counting_iterator<int, thrust::host_space_tag> stripe_idx_it(0);
+  thrust::counting_iterator<int, thrust::host_system_tag> col_idx_it(0);
```

Comment:
Why is the system tag being specified at all?

Reply:
Looking at how these are used, these tags are unnecessary and can just be removed all together.

Reply:
Hah, I thought that these tags are required. Yeah, then just `thrust::counting_iterator<int>`.

Reply:
Removed! I wondered the same thing but wasn't sure. :)

Reply:
I verified and this was the only place that a tag is used.